rush-plugins/rush-azure-storage-build-cache-plugin/package.json

```diff
@@ -19,6 +19,7 @@
   },
   "dependencies": {
     "@azure/identity": "~2.1.0",
+    "@azure/keyvault-secrets": "~4.6.0",
```

Comment:
This isn't explicitly useful to someone who is using this package for its primary purpose (as a Rush cache provider). It would be preferable to include this library in whatever code would be making use of this functionality.